python/ray/util/scheduling_strategies.py

```diff
@@ -72,9 +72,90 @@ def __init__(
         self._fail_on_unavailable = _fail_on_unavailable
 
 
+def _validate_values(values):
+    if values is None or len(values) == 0:
+        raise ValueError("The value of lable cannot be empty.")
+
+    for value in values:
+        if not isinstance(value, str):
+            raise ValueError("The value of lable must be string.")
```

Comment:
Be a little more specific here like `Expected value {value} in position {index} to be of type str`

Reply:
Fixed